Parameters.java

```diff
@@ -84,14 +84,17 @@ public String getValidCharacters() {
 	// parameters specific to GeometricPhenotype
 	public static double smithConversion = 0.1;					// multiplier to distance to give cross-immunity
 	public static double homologousImmunity = 0.05;				// immunity raised to antigenically identical virus
-	public static double initialTraitA = -6;
-	public static double meanStep = 0.3;
-	public static double sdStep = 0.3;
+	public static double initialTraitA = -6;					// value in dimension 1 for initial host immunity
+	public static double meanStep = 0.3;						// mean mutation size for non-epitopes
+	public static double sdStep = 0.3;							// standard deviation of mutation size for non-epitopes
+	public static double meanStepEpitope = 0.3;					// mean mutation size for epitopes
+	public static double sdStepEpitope = 0.3;					// standard deviation of mutation size for epitopes
+	public static int[] epitopeSites = {};						// epitope sites of Virus (valid inputs are between 1 and startingSequence.length() / 3)
```

Comment:
lower-case "v" for Virus

Reply:
I use upper-case "V" for Virus objects. For this line, "v" would make more sense. Thanks!